src/app/content/highlights/components/CardWrapper.spec.tsx

```diff
@@ -1,25 +1,47 @@
 import React from 'react';
+import { Provider } from 'react-redux';
 import renderer from 'react-test-renderer';
+import createTestStore from '../../../../test/createTestStore';
 import createMockHighlight from '../../../../test/mocks/highlight';
+import { Store } from '../../../types';
 import Card from './Card';
 import CardWrapper from './CardWrapper';
 
 jest.mock('./Card', () => () => null);
+jest.mock('./cardUtils', () => ({
+  getHighlightTopOffset: () => 0,
+}));
 
 describe('CardWrapper', () => {
-  it('matches snapshot', () => {
-    const component = renderer.create(<CardWrapper
-      highlights={[createMockHighlight()]}
-    />);
+  let store: Store;
+
+  beforeEach(() => {
+    store = createTestStore();
+  });
+
+  it('matches snapshot', async() => {
+    const component = renderer.create(<Provider store={store}>
```

Comment:
Is store needed to test the wrapper itself? Doesn't look like the tests depend on store doing anything besides passing the same value so you could pass it by props.